models/conversation.js

```diff
@@ -0,0 +1,34 @@
+let mongoose = require('mongoose');
+
+// Conversation Schema
+let ConversationSchema = mongoose.Schema({
+    patientName :{
```

Comment:
```suggestion
    patientName: {
```
please also be consistent with this for the others fields